### PR TITLE
CU-295d6bv - Add partners grouping by tier

### PR DESCRIPTION
**Pull request by pmakowski23**

### CODE REVIEW

**Comment by pmakowski23** [comment]
Task linked: [CU-295d6bv Add partners grouping by tier](https://app.clickup.com/t/295d6bv)

File: components/sponsors/sponsors-grid.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ function SponsorCard({ sponsor }: { sponsor: Sponsor }) {
   return (
     <Link key={sponsor.name} href={sponsor.website}>
       <a
-        target='_blank'
+        target="_blank"
```
Comment (stepniaczky) [inline components/sponsors/sponsors-grid.tsx:11]: huge

**Review by stepniaczky — APPROVED** [review]
legit as always

**Review by novacooo — APPROVED** [review]
huge legit

**Pull request merged**